pkgdown ymlのBugを修正する。

```
Error in install.packages("pkgdown", type = "binary") : 
  type 'binary' is not supported on this platform
Execution halted
Error: Process completed with exit code 1.
```
binaryが問題なのか？

--- a/.github/workflows/pkgdown.yaml
+++ b/.github/workflows/pkgdown.yaml
@@ -39,7 +39,7 @@ jobs:
       - name: Install dependencies
         run: |
           remotes::install_deps(dependencies = TRUE)
-          install.packages("pkgdown", type = "binary")
+          install.packages("pkgdown")
         shell: Rscript {0}
 
       - name: Install package

